src/gl.rs

```diff
@@ -248,6 +248,30 @@ pub fn bind_attrib_location(program: GLuint, index: GLuint, name: &str) {
     }
 }
 
+#[cfg(not(target_os="android"))]
+#[inline]
+pub fn get_uniform_block_index(program: GLuint, name: &str) -> GLuint {
+    unsafe {
+        ffi::GetUniformBlockIndex(program, CString::new(name).unwrap().as_ptr())
```

Comment:
Hmm, I did wonder that when I copy/pasted it from other bindings in this file... I'll fix them all up.